core/templates/components/forms/custom-forms-directives/image-uploader.component.ts

```diff
@@ -50,7 +50,7 @@ export class ImageUploaderComponent {
   licenseUrl = AppConstants.PAGES_REGISTERED_WITH_FRONTEND.LICENSE.ROUTE;
 
   constructor(
-    private blogDashboardPageService: BlogDashboardPageService,
+    public blogDashboardPageService: BlogDashboardPageService,
```

Comment:
Here and everywhere else for this private to public change:
This is done because we use the service in the template. In the angular AoT compilation, the angular compiles the template to TS (and then JS) instructions and does type checking. private variables are accessible in the templates.